en/guide/migrating-5.md

```diff
@@ -116,6 +117,14 @@ The `res.sendfile()` function has been replaced by a camel-cased version `res.se
 
 The `app.router` object, which was removed in Express 4, has made a comeback in Express 5. In the new version, this object is a just a reference to the base Express router, unlike in Express 3, where an app had to explicitly load it.
 
+<h4 id="path-to-regexp"><path-to-regexp</h4>
+
+From Express 5 onwards, following are the changes with respect to path-to-regexp which has direct implications to route.
+* RegExp special characters can now only be used in a parameter, as opposed to anywhere.
+* Parameters can have suffixes to augment meaning - *, + and ?. E.g. /:user*
```

Comment:
Looks like you need to escape or use code for the markdown special characters, they're currently italicizing the text.

Reply:
@blakeembrey, thanks. Used code for special characters. PTAL.